.gitlab-ci.yml

```diff
@@ -35,72 +39,121 @@ build-simnetscripts-image-and-push-temporarly-in-paritypr:
         quay.io/buildah/stable \
             buildah bud  \
               --layers=true  \
-              --cache-from  "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA" \
-              --tag         "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA" \
+              --cache-from  "${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG}" \
+              --tag         "${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG}" \
               /build 
-      echo "LOG INFO step 5 image $CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA built successfully"
+      echo "LOG INFO image ${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG} built successfully"
       echo "$DOCKER_SIMNET_PASS" \
            | podman  login --username "$DOCKER_SIMNET_USER" --password-stdin docker.io
-      podman push --format=v2s2 "$CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA"
-      echo "LOG INFO step 6 image $CONTAINER_IMAGE:$CI_COMMIT_SHORT_SHA pushed successfully"
+      podman push --format=v2s2 "$CONTAINER_IMAGE:$CONTAINER_IMAGE_TAG"
+      echo "LOG INFO ${CONTAINER_IMAGE}:${CONTAINER_IMAGE_TAG} pushed successfully"
       echo "LOG check if need to cleanup dangling images"
       df -h | grep /var/lib/containers
       CACHE_USE_PERCENT="$(df -h | grep -vE '/var/lib/containers/.*' | grep /var/lib/containers | awk '{print $5}' | sed 's/%//g')"
-      test -n "$CACHE_USE_PERCENT" \
+      test -n "${CACHE_USE_PERCENT}" \
               || ( echo "This variable can't be emtpy" ; exit 1 )
-      if test "$CACHE_USE_PERCENT" -ge 80 ; then 
-        echo "LOG INFO need to cleanup cache, value: $CACHE_USE_PERCENT"
+      if test "${CACHE_USE_PERCENT}" -ge 80 ; then 
+        echo "LOG INFO need to cleanup cache, value: ${CACHE_USE_PERCENT}"
         time podman rmi -f $(podman images -q -f "dangling=true")
       else 
-        echo "LOG INFO No Cleanup needed this time value: $CACHE_USE_PERCENT"
+        echo "LOG INFO No Cleanup needed this time value: ${CACHE_USE_PERCENT}"
       fi
   retry: 1
   after_script:
     - podman logout docker.io
 
+test-current-latest-container-image:
+  stage:                           build
+  tags:
+    - parity-simnetscripts
+  variables:
+    IMAGE_UNDER_TEST:              "paritytech/simnetscripts"
+    IMAGE_UNDER_TEST_TAG:          "latest"
+    # use parity/polkadot as default until we will have paritypr/synth-wave:master
+    PARACHAINS_IMAGE:              "docker.io/parity/polkadot"
+    # PARACHAINS_IMAGE:              "docker.io/paritypr/synth-wave"
+    # there is no master tag for docker.io/paritypr/synth-wave
```

Comment:
now we have [paritypr/synth-wave:master](https://hub.docker.com/layers/paritypr/synth-wave/master/images/sha256-8884021e6ae719c93829ce2c407835e67bf2f278939bd20664ce5d9dda23e10f?context=repo)
It's being pushed from [every master](https://github.com/paritytech/polkadot/pull/2885/files#diff-037ea159eb0a7cb0ac23b851e66bee30fb838ee8d0d99fa331a1ba65283d37f7R270) change.